lib/main.dart

```diff
@@ -7,109 +7,274 @@ void main() {
+                  SizedBox(
+                    width: 300,
+                    height: 50,
+                    child: TextField(
+                      obscureText: true,
+                      decoration: InputDecoration(
+                        border: OutlineInputBorder(
+                            borderRadius: BorderRadius.circular(25)),
+                        labelText: 'Новая публикация',
+                      ),
+                    ),
+                  )
+                ],
+              ),
+              const Padding(padding: EdgeInsets.only(top: 10)),
+              Row(
+                children: [
+                  TextButton.icon(
+                      onPressed: () {},
+                      icon: Icon(Icons.photo),
+                      label: Text('Фото')),
+                  TextButton.icon(
+                      onPressed: () {},
+                      icon: Icon(Icons.play_circle_fill),
+                      label: Text('Видео')),
+                  TextButton.icon(
+                      onPressed: () {},
+                      icon: Icon(Icons.calendar_today),
+                      label: Text('Мероприятие')),
+                ],
+              ),
+              Row(
+                children: [
+                  TextButton.icon(
+                      onPressed: () {},
+                      icon: Icon(Icons.article),
+                      label: Text('Написать статью')),
+                ],
+              )
+            ],
+          )
+        ],
+      ),
+    );
   }
+}
+
+class AddPeople extends StatelessWidget {
+  const AddPeople({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    // This method is rerun every time setState is called, for instance as done
-    // by the _incrementCounter method above.
-    //
-    // The Flutter framework has been optimized to make rerunning build methods
-    // fast, so that you can just rebuild anything that needs updating rather
-    // than having to individually change instances of widgets.
-    return Scaffold(
-      appBar: AppBar(
-        // Here we take the value from the MyHomePage object that was created by
-        // the App.build method, and use it to set our appbar title.
-        title: Text(widget.title),
+    return Container(
+      color: const Color(0xFEFEFEFE),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Column(
+            children: [
+              const Padding(padding: EdgeInsets.only(top: 20)),
+              const Text(
+                'Nikita, хотите получать еще больше \n обновлений?',
+                style: TextStyle(
+                  fontSize: 20,
+                ),
+              ),
+              const Padding(padding: EdgeInsets.only(top: 10)),
+              const Text(
+                  'Добавьте людей, которых вы можете знать, в свою сеть \n контактов и просматривайте их обновления в ленте.'),
```

Comment:
ideally, you shouldn't make your text multiline like this, on bigger screen you might want to show it in a single line. Your strings should be UI-agnostic (without `\n` in simple words) and your widgets should control how many lines this text will use